### DISCUSSION TITLE
How do I fix this error?

### DISCUSSION THREAD

**Discussion by Herblawunf** [neoclide/coc.nvim#4388] [Q&A] [answered]
- ERROR: Javascript entry not found, please compile coc.nvim by esbuild.
  - ERROR: service could not be initialized
  
  I am getting this error when I run the checkhealth command. I cannot find my coc.nvim directory, where would this be and why am i getting this error?
  Thanks for the help!

**Comment by chemzqm** [comment, marked as answer]
Use release without the need to build the source code, https://github.com/neoclide/coc.nvim/wiki/Install-coc.nvim#add-cocnvim-to-vims-runtimepath

**Reply by Herblawunf** [reply]
I am already using that though

**Reply by chemzqm** [reply]
It's broken, remove the coc.nvim folder and install again.

**Reply by Herblawunf** [reply]
That's the problem, there is no folder, I used the plugin to download it and I can;t find the folder in my .vim folder

**Reply by chemzqm** [reply]
Checkout your runtimepath by `:echo &rtp`
